fal/controllers/bench_swaps.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+from typing import List, Tuple, TYPE_CHECKING
+import io
+import http.client
+import re
+import gzip
+import configparser
+
+from fal.orm import Secret, mfalncfm_main
+from fal.models import Team, Season, Anime
+
+if TYPE_CHECKING:
+    from sqlalchemy.orm import Session
+
+httpconn = http.client.HTTPSConnection("myanimelist.net")
```

Comment:
I think we should use [requests](https://requests.kennethreitz.org/en/master/) as it's way easier to use than the `http.client` library.

Reply:
Probably. We should probably rewrite all this logic. I just lifted this from old script and ran 2to3 on it so we can get going quick. I tried to isolate it so we can easily replace it moving forward.